ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutor.java

```diff
@@ -237,7 +277,7 @@ private static WhereInfo extractWhereInfo(
 
     if (!windowed) {
       if (comparisons.size() > 1) {
-        throw invalidWhereClauseException("Unsupported WHERE clause", windowed);
+        throw invalidWhereClauseException("Unsupported WHERE clause", false);
```

Comment:
Inconsistent case compare with preceeding "missing WHERE clause". Either Unsupported and Missing or unsupported and missing.

Reply:
In some cases the user hasn't supplied a WHERE clause, so it's missing.
In other cases the user has supplied a WHERE clause but it contains things we don't support, so it's unsupported.  

These are not inconsistent... they are different!

Reply:
Ah, you mean the capitalisation of the error message!